### DISCUSSION TITLE
Welcome to Rath Discussions!

### DISCUSSION THREAD

**Discussion by ObservedObserver** [Kanaries/Rath#255] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ssaurbier** [comment]
Hey there, my name is steven saurbier and i'm a budding ds (whatever that means) and data analyst. I have extensive experience in the crypto space since 2012, which may or may not be relevant, as well as general interest in decentralized data management and structuring, incentivization, and monetization. 

All that aside, how can I help implement this feature within a notebook environment (eg juypter)?

**Reply by ObservedObserver** [reply]
Rath is basically a react app in the frontend, so the easiest way might be using an iframe to embed it.

It is also possible to use it like a react component, but it needs a little more work. Not quite sure using `jupyter-react` can make it happen (may require some works on webpack loaders). The sub-module [graphic-walker](https://github.com/Kanaries/graphic-walker), implementation of manual analysis, has already been built into a react component, so we can test it first.

I need to do some research on it later. It is a great idea to use Rath in juypter.
